MEA.py

```diff
@@ -12776,16 +12767,16 @@ def mass_scan(f_path) :
 			
 			# ME4-Only Fix 3 : The usual method to detect EXTR vs RGN does not work for ME4, KRND. not enough
 			if fw_type_fix :
-				effs_match = (re.compile(br'\x45\x46\x46\x53\x4F\x53\x49\x44')).search(reading) # EFFSOSID detection
+				effs_match = (re.compile(br'EFFSOSID')).search(reading) # EFFSOSID detection
 				if effs_match is not None :
 					(start_effs_match, end_effs_match) = effs_match.span()
 					effs_start = int.from_bytes(reading[end_effs_match:end_effs_match + 0x4], 'little')
 					effs_size = int.from_bytes(reading[end_effs_match + 0x4:end_effs_match + 0x8], 'little')
 					effs_data = reading[fpt_start + effs_start:fpt_start + effs_start + effs_size]
 				
-					me4_type_fix1 = (re.compile(br'\x4D\x45\x5F\x43\x46\x47\x5F\x44\x45\x46')).findall(effs_data) # ME_CFG_DEF detection (RGN have 2-4)
-					me4_type_fix2 = (re.compile(br'\x47\x50\x49\x4F\x31\x30\x4F\x77\x6E\x65\x72')).search(effs_data) # GPIO10Owner detection
-					me4_type_fix3 = (re.compile(br'\x41\x70\x70\x52\x75\x6C\x65\x2E\x30\x33\x2E\x30\x30\x30\x30\x30\x30')).search(effs_data) # AppRule.03.000000 detection
+					me4_type_fix1 = (re.compile(br'ME_CFG_DEF')).findall(effs_data) # ME_CFG_DEF detection (RGN have 2-4)
+					me4_type_fix2 = (re.compile(br'GPIO10Owner')).search(effs_data) # GPIO10Owner detection
+					me4_type_fix3 = (re.compile(br'AppRule.03.000000')).search(effs_data) # AppRule.03.000000 detection
```

Comment:
An escape for each dot character is required here, otherwise they match everything except new line

Reply:
ack